### DISCUSSION TITLE
how to make a continuous animation in the trips

### DISCUSSION THREAD

**Discussion by parkjk93** [visgl/deck.gl#5694] [Q&A]
Hi,
I am making an example [TripsLayer.](https://github.com/visgl/deck.gl/tree/8.4-release/examples/website/trips) 
set loopLength 1800 as in the example.
when the animation time goes from 1800 to 0, Even if there is traill time left, the whole turns off and then turns on again.
like this, (red circle means time)
![image](https://user-images.githubusercontent.com/38125895/115824083-5ec99c80-a442-11eb-97fb-24a23183a9fa.png)
![image](https://user-images.githubusercontent.com/38125895/115824092-61c48d00-a442-11eb-9e18-e496ee49f12a.png)

but, in this [Example](https://deck.gl/examples/trips-layer/) Seems to be continuous

how can i make it continuous?

**Comment by sclebal** [comment]
Hi @parkjk93, the example isn't continuous, it's just slow. If you set `animationSpeed` with a higher value, for example 10, you can see that after a while the animation restart from the beginning.

![Screen Recording 2021-04-26 at 17 45 36](https://user-images.githubusercontent.com/9151432/116111922-5e860700-a6b7-11eb-946b-629a918f1568.gif)

Hope it helps!
